share/ext/luajit/build_luajit.sh

```diff
@@ -225,6 +225,8 @@ case $1 in
 			make -j8
 			make install
 			mv $PREFIX/bin/$CONF_TARGET/${TARGET_FILE} $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
+			# remove symlink. we don't need it, and it doesn't extract under git-bash
+			rm -v $PREFIX/bin/$CONF_TARGET/luajit
```

Comment:
Not sure if I've manually removed these files from the packages, or if they've started showing with the new version.
In any case, Git Bash doesn't handle extracting symlinks, so we need to make sure to remove them.

Yes, we currently extract all host packages even if it's not the target platform.
It makes the scripts _so_ much easier to maintain in terms of cross compiling.